test/run/test_noop.py

```diff
@@ -34,6 +34,7 @@
 }
 
 
+@pytest.mark.skipif(not test.TestBase.can_bind_mount(), reason="root-only")
```

Comment:
This test has not been converted to `pytest` yet; there is `@unittest.skipUnless(test.TestBase.can_bind_mount(), "root-only")` so you don't need to import `pytest`; or you could also convert the test to use pytest instead of `unittest.TestCase`.

Reply:
If you are tempted by the conversion, here is an example where it has been done before: 8cf35b4728c7bc5cdb8097344657c06b372dd41d

Reply:
Thanks for the review. Alright, The conversion looks interesting, I'm gonna have a look.